enlighten/common.py

```diff
@@ -134,3 +134,44 @@ def get_default_data_dir():
     if os.name == "nt":
         return os.path.join(os.path.expanduser("~"), "Documents", "EnlightenSpectra")
     return os.path.join(os.environ["HOME"], "EnlightenSpectra")
+
+
+"""
+Looking for a method to display Marquee messages? 
+(Those non-intrusive messages at the top of Enlighten)
+
+One way is to use self.ctl.marquee.info("Message")
+This assumes your FeatureObject class has an instance of Controller.
+"""
+
+def msgbox(prompt, title="Alert", buttons=0):
+    """
+    Display an interupting message to the user.
+
+    Inspired by VB msgbox: 
```

Comment:
> Inspired by VB

🤮 (but I get it :-)

Reply:
I'm breaking my own rule of not borrowing culture from other languages, but I remember my first day on Enlighten, I wanted nothing more than to say MSGBOX !

I think it's pythonic enough,,, except the buttons parameter taking an int like the original VB might not be worth keeping around. Maybe we do away with the button parameter and have msgbox always give only an OK button.

Then we can borrow from javascript and use different functions for different alert box behavior: 
msgconfirm() --> Ok/Cancel
msgprompt() --> Yes/No